Music app/app/src/main/java/com/example/musicapp/Activities/PlayerActivity.java

```diff
@@ -252,99 +218,63 @@ public void run()
         nextThread.start();
     }
 
-    private void nextBtnClicked()
+    private void nextBtnSupp()
```

Comment:
Why this function is needed?

Reply:
The same answer as a question for prevBtnSupp method.